smart_control/simulator/building_radiation_utils_test.py

```diff
@@ -0,0 +1,296 @@
+                [ -1,  -2,  -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+                [ -1,  -2,  -3,  -3,  -3,  -3,  -3,  -3,  -3,  -3,  -2,  -1],
+                [ -1,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -1],
+                [ -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1]])
+    # fmt: on
+    # pylint:enable=line-too-long
+    result, _ = mark_air_connected_interior_walls(
+        indexed_floor_plan=indexed_floor_plan,
+        start_pos=(2, 3),
+        interior_wall_value=constants.INTERIOR_WALL_VALUE_IN_FUNCTION,
+        marked_value=-33,
+        air_value=constants.INTERIOR_SPACE_VALUE_IN_FUNCTION,
+    )
+    # setup (temperatures and IFAinv have same number of rows):
+    self.assertEqual(result.shape, indexed_floor_plan.shape)
+
+    # result has same shape as the temperatures array:
+    with self.subTest("result as expected"):
+      assert_array_almost_equal(result, expected_result)
+
+  def test_mark_directly_seeing_nodes(self):
+    # fmt: off
+    #pylint:disable=line-too-long
+
+    indexed_floor_plan =\
+       np.array(
+        [[ -1, -1, -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1],
+        [ -1,  -2, -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -1],
+        [ -1,  -2, -3,  -3,  -3,  -3,  -3,  -3,  -3,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,  -3,  -3,  -3,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,  -3,   0,  -3,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,  -3,   0,  -3,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,  -3,   0,  -3,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,  -3,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,  -3,  -3,  -3,  -3,  -3,  -3,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,  -3,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,  -3,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,  -3,  -3,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,   0,   0,   0,   0,   0,   0,  -3,  -2,  -1],
+        [ -1,  -2, -3,  -3,  -3,  -3,  -3,  -3,  -3,  -3,  -2,  -1],
+        [ -1,  -2, -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -1],
+        [ -1,  -1, -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1]]
+      )
+
+
+
+    expected_result_23 = \
+      np.array([[ -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1],
+       [ -1,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -2,  -1],
+       [ -1,  -2,  -3, -67, -34, -34, -34, -34, -34,  -3,  -2,  -1],
```

Comment:
We have a question about the meaning of these values: for example, what does -67 represent and what does -34 represent? 

It would be nice to have some documentation about this. Apologies if we overlooked this somewhere. Perhaps we can set some constant values in the code.